slack-archive/src/main/java/juja/microservices/slack/archive/model/dto/ChannelDTO.java

```diff
@@ -0,0 +1,25 @@
+package juja.microservices.slack.archive.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.*;
+
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "id",
+        "name"
+})
+@Getter
```

Comment:
Maybe @Data instead of many Annotations?

Reply:
done